### DISCUSSION TITLE
blog/how-to-build-a-code-editor-with-codemirror-6-and-typescript/introduction

### DISCUSSION THREAD

**Discussion by giscus** [davidmyersdev/website#12] [Comments]
# blog/how-to-build-a-code-editor-with-codemirror-6-and-typescript/introduction

Stay current in the evolving world of web development.

https://davidmyers.dev/blog/how-to-build-a-code-editor-with-codemirror-6-and-typescript/introduction

**Comment by pappasa** [comment]
Great article, thank you! Have you managed to continue the series, and if yes, where can I find it?

**Reply by davidmyersdev** [reply]
Thanks so much, @pappasa! I have not released the next part yet. I've been focused on work and other priorities lately, but I really hope to come back to this soon. Is there anything in particular you'd be interested in learning more about?

**Comment by pappasa** [comment]
I would be interested to see how to build a code editor with intellisense, that means the editor should suggest the properties of an object, and the parameters of a function, like VSCode does it.

**Comment by kimheffy** [comment]
Amazing article! You have demystified a lot of my confusions especially around @codemirror imports and codemirror. Also I have seen plenty of examples on CodeMirror, all of which were pretty confusing.

I'm wondering how you managed to add linters to your extensions?

**Comment by jangxyz** [comment]
Very nice. It took 15 minutes to explain what took me hours to figure out by manually poking here and there. 

This powerful library can really have some help on the documentation, which will help a lot of people from suffering. I see that you'be been busy, but if you are still interested, I would really like to understand how the internal systems works -- with all the StateField, StateEffect, ViewPlugin, Widget, `~.of` and stuff.
